src/ui/view/vault.rs

```diff
@@ -184,15 +184,15 @@ impl VaultOnChainTransactionsPanel {
                         Row::new()
                             .push(
                                 Container::new(text::simple(
-                                    "This vault needs to be acknowledged ",
+                                    "Do you want to create of vault from this deposit ? ",
                                 ))
                                 .width(Length::Fill),
                             )
                             .push(
                                 Container::new(
                                     button::important(
                                         &mut self.action_button,
-                                        button::button_content(None, "Acknowledge"),
+                                        button::button_content(None, "Create Vault"),
```

Comment:
Sometimes we use the capital V for vault and sometimes we don't, we should be consistent. I think I prefer the lowercase version: "Create vault"